webpack.config.js

```diff
@@ -4,6 +4,7 @@ var webpack = require('webpack')
 var ip = process.env.IP || '0.0.0.0'
 var port = process.env.PORT || 3000
 var DEBUG = process.env.NODE_ENV !== 'production'
+var HtmlWebpackPlugin = require('html-webpack-plugin');
```

Comment:
Please, place this line at the top (after other requires) and remove the semicolon.